tests/library/page-close.spec.ts

```diff
@@ -153,7 +153,7 @@ test('interrupt request.response() and request.allHeaders() on page.close', asyn
   await page.close();
   expect((await respPromise).message).toContain(kTargetClosedErrorMessage);
   // All headers are the same as "provisional" headers in Firefox.
-  if (browserName === 'firefox')
+  if (browserName === 'firefox' || browserName as any === '_bidiFirefox')
```

Comment:
Is `as any` really needed? In other tests you aren't using it.

Reply:
For some reason`browserName` includes the BiDi browsers in tests under `tests/pages/`, but not in tests under `tests/library/`.

Reply:
In the `tests/page` tests we extend public playwright fixtures, where browserName is already defined and has only 3 values. Fixtures can be overridden but their type must stay the same. In the `tests/library` tests we use completely custom fixtures and the type contains `__bidiFirefox`. I think it's okay to do `as any` for the expectations here.